input/vulnerabilities/One_class_to_rule_them_all.json

```diff
@@ -53,5 +53,11 @@
     , "commit" : "8d57b9dbbd883422a0ff02083bfcf637b097e504"
     , "component" : "platform/external/conscrypt"
     }
-  }
+  },
+  "Surface": ["local", "app"],
+  "Vector": ["kernel-vulnerability"],
```

Comment:
I don't think this is a kernel vulnerability but a vulnerability which allows arbitrary kernel modules to be loaded from system privilege. We might want to distinguish between "root" and "kernel" privilege as with SELinux root can't do everything but if you are running arbitrary kernel code then it is game over.

Reply:
The most recent commit of that file has `daemon-abusing` instead of `kernel-vulnerability` there but I'm happy to update other bits (e.g. do you mean it would be good to have `kernel` as an additional item on the list of possible privilege levels) if necessary?